Doc/library/io.rst

```diff
@@ -960,22 +961,25 @@ Text I/O
 
 .. class:: StringIO(initial_value='', newline='\\n')
 
-   An in-memory stream for text I/O.  The text buffer is discarded when the
-   :meth:`~IOBase.close` method is called.
+   A text stream using an in-memory text buffer.  It inherits
+   :class:`TextIOBase`.
+   
+   The text buffer is discarded when the :meth:`~IOBase.close` method is
+   called.
 
    The initial value of the buffer can be set by providing *initial_value*.
    If newline translation is enabled, newlines will be encoded as if by
    :meth:`~TextIOBase.write`.  The stream is positioned at the start of
    the buffer.
 
-   The *newline* argument works like that of :class:`TextIOWrapper`.
-   The default is to consider only ``\n`` characters as ends of lines and
-   to do no newline translation.  If *newline* is set to ``None``,
-   newlines are written as ``\n`` on all platforms, but universal
-   newline decoding is still performed when reading.
+   The *newline* argument works like that of :class:`TextIOWrapper`,
+   except that if *newline* is set to ``None``, newlines are written as
+   ``\n`` on all platforms, but universal newline decoding is still
+   performed when reading.  The default is to consider only ``\n``
+   characters as ends of lines and to do no newline translation.
```

Comment:
Does newline translation happen during reading? If so, it might be good to group together both things that happen during reading in one sentence and then the writing behavior in another sentence.

Reply:
Yes universal newline translation happens during reading if the `newline` parameter is to `None`. Alright, let's split the reading behavior from the writing behavior to be clearer.